include/oneapi/dpl/internal/dynamic_selection_impl/scoring_policy_defs.h

```diff
@@ -0,0 +1,31 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _ONEDPL_SCORING_POLICY_DEFS_H
+#define _ONEDPL_SCORING_POLICY_DEFS_H
+
+#include "oneapi/dpl/internal/dynamic_selection_traits.h"
+namespace oneapi {
+namespace dpl {
+namespace experimental {
+  template<typename Policy, typename Resource>
+  class basic_selection_handle_t {
+    Policy p_;
+    Resource e_;
```

Comment:
Do we consider policies and resources as "common reference" types, so that copying those into a selection handle is appropriate?

Reply:
Yes.